tlscanary/modes/basemode.py

```diff
@@ -62,6 +62,10 @@ def setup_args(cls, parser):
                            default=base_default)
 
         group = parser.add_argument_group("profile setup")
+        group.add_argument("-c", "--cache",
+                           help='Allow profiles to cache web content',
+                           action="store_true",
+                           default=False)
```

Comment:
I definitely messed this up before myself: `store_true` implies `default=False`, making it redundant. No need to change, just FYI and note so self for a future cleanup.